delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/Projects.scala

```diff
@@ -183,32 +184,25 @@ object Projects {
     }
 
   private[delta] def evaluate(
-      fetchOrg: Label => UIO[Option[OrganizationResource]]
+      fetchOrg: Label => IO[ProjectRejection, Organization]
   )(state: ProjectState, command: ProjectCommand)(implicit
       clock: Clock[UIO] = IO.clock,
       uuidF: UUIDF
   ): IO[ProjectRejection, ProjectEvent] = {
 
-    def validateOrg(label: Label) =
```

Comment:
I think this is dangerous... because we want the logic that projects should not be modified if the org is deprecated to be in the state machine, not outside

Reply:
Yes, I totally agree. For some reason I forgot this